v63003/outref/gtm8182.txt

```diff
@@ -1,27 +1,166 @@
+# that GDE CHANGE -INSTANCE -FILE_NAME= "" removes the .repl mapping in the gld file.
+----------------------------------------------------------------------------
+
+# Show INST1 mapped instance file
+
+# Start INST1 INST2 replication
 ==Executing MULTISITE_REPLIC 'START INST1 INST2'==
 Starting Primary Source Server in ##TEST_PATH##
 Starting Passive Source Server and Receiver Server in ##FILTERED##_REMOTE_TEST_PATH_/instance2
 
+# Start INST3 INST4 replication
 ==Executing MULTISITE_REPLIC 'START INST3 INST4'==
 Starting Primary Source Server in ##FILTERED##_REMOTE_TEST_PATH_/instance3
 Starting Passive Source Server and Receiver Server in ##FILTERED##_REMOTE_TEST_PATH_/instance4
 
+# Remove and replace INST3 .repl mapping 
+# to test that .repl file mapping overrides the gtm_repl_instname env var.
+----------------------------------------------------------------------------
+# Show INST3 mapped instance file
+##TEST_AWK.*/instance3/mumps.repl
+# Remove INST3 mapping
+# Change INST3 mapped instance file
+# Show INST3 mapped instance file
+##TEST_AWK.*/instance3/mumps.repl
+
+
 # Run gtm8182.m to update both INST1 and INST3 source server
+----------------------------------------------------------------------------
     Update the INST1 DB
+    Update the INST3 DB with an extended reference
     Switch to INST3 gbldir
     Update the INST3 DB
+    Update the INST1 DB with an extended reference
     Switch back to INST1 gbldir
+
+# Show INST1 mapped instance file
+
+# Sync originating and replicating instances
+==Executing MULTISITE_REPLIC 'SYNC INST1 INST2'==
+==Executing MULTISITE_REPLIC 'SYNC INST3 INST4'==
+
 # Check INST2 receiver server for update (expecting only ^jake to be defined)
 ==Executing MULTISITE_REPLIC 'RUN INST2 $MUPIP extract INST2_extract.glo'==
 %YDB-I-RECORDSTAT, ^jake:	  Key cnt: 1  max subsc len: 6  max rec len: 8  max node len: 18
-%YDB-I-RECORDSTAT, TOTAL:	  Key cnt: 1  max subsc len: 6  max rec len: 8  max node len: 18
+%YDB-I-RECORDSTAT, ^wendy:	  Key cnt: 1  max subsc len: 7  max rec len: 6  max node len: 17
+%YDB-I-RECORDSTAT, TOTAL:	  Key cnt: 2  max subsc len: 7  max rec len: 8  max node len: 18
 
 # Check INST4 receiver server for update (expecting only ^zack to be defined)
 ==Executing MULTISITE_REPLIC 'RUN INST4 $MUPIP extract INST4_extract.glo'==
+%YDB-I-RECORDSTAT, ^jim:	  Key cnt: 1  max subsc len: 5  max rec len: 6  max node len: 15
+%YDB-I-RECORDSTAT, ^zack:	  Key cnt: 1  max subsc len: 6  max rec len: 6  max node len: 16
+%YDB-I-RECORDSTAT, TOTAL:	  Key cnt: 2  max subsc len: 6  max rec len: 6  max node len: 16
+
+
+# Remove mapping from both INST1 and INST3 in order to test for error
+# upon using both instances in a single process
+----------------------------------------------------------------------------
+# Remove INST3 mapping
+# Remove INST1 mapping
+# Run gtm8182.m again and search output for error
+##TEST_AWK^%YDB-E-REPLINSTMISMTCH.*
```

Comment:
I would like the full error message text here. This is a test that the error message content is correct and identifies the appropriate instance and database file names.

Reply:
I still don't see the full error message text here. It is TEST_AWKed out. Will wait for this to be addressed before reviewing further.